rules/REQUEST-932-APPLICATION-ATTACK-RCE.conf

```diff
@@ -1244,6 +1244,44 @@ SecRule REQUEST_COOKIES|!REQUEST_COOKIES:/__utm/|REQUEST_COOKIES_NAMES|ARGS_NAME
     setvar:'tx.rce_score=+%{tx.critical_anomaly_score}',\
     setvar:'tx.inbound_anomaly_score_pl2=+%{tx.critical_anomaly_score}'"
 
+# [ Unix shell expressions - Bash Tilde expansion ]
+#
+# Detects the following patterns which are common in Unix shell scripts
+# and one-liners:
+#
+# ~+    $PWD
+# ~-    $OLDPWD
+# ~4    fourth directory entry on the stack from the top
+# ~-2   second directory entry on the stack from the top
+# ~+2   second directory entry on the stack from the bottom
+#
+# Reference - https://linuxsimply.com/bash-scripting-tutorial/expansion/tilde-expansion/
+#
+# Regular expression generated from regex-assembly/932271.ra.
+# To update the regular expression run the following shell script
+# (consult https://coreruleset.org/docs/development/regex_assembly/ for details):
+#   crs-toolchain regex update 932271
+#
```

Comment:
Add a "sibling" comment, e.g. `# This is a sibling of rule 932271` and add a sibling comment to the base rule: `# This rule has a stricter sibling: 932271`